app/src/main/java/io/gnosis/safe/notifications/NotificationRepository.kt

```diff
@@ -260,7 +280,8 @@ class NotificationRepository(
     }
 
     private fun generateUUID(): String {
-        return UUID.randomUUID().toString().toLowerCase()
+        deviceUuid = UUID.randomUUID().toString().toLowerCase()
+        return deviceUuid!!
```

Comment:
why do we need !! here?

Reply:
Because the type of `deviceUuid` is nullable.

Reply:
but not the generated string from UUID?